### PR TITLE
Fix python linter errors for the qa/ directory

### PR DESCRIPTION
**Pull request by JanuszL**
- fixes python linter errors for the qa/ directory
- adds the qa/ directory to the linter scan

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

## Category:
**Other** (*e.g. Documentation, Tests, Configuration*)

## Description:
- fixes python linter errors for the qa/ directory
- adds the qa/ directory to the linter scan

### Affected modules and functionalities:
- linter cmake
- qa/setup_packages.py

### Key points relevant for the review:
- NA

### Tests:

- [ ] Existing tests apply
- [ ] New tests added
  - [ ] Python tests
  - [ ] GTests
  - [ ] Benchmark
  - [ ] Other
- [x] N/A

### Documentation
- [ ] Existing documentation applies
- [ ] Documentation updated
  - [ ] Docstring
  - [ ] Doxygen
  - [ ] RST
  - [ ] Jupyter
  - [ ] Other
- [x] N/A

#### Requirements
- [ ] Implements new requirements
- [ ] Affects existing requirements
- [x] N/A

**REQ IDs**: N/A

**JIRA TASK**: N/A

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

**Comment by JanuszL** [comment]
!build

File: qa/setup_packages.py [hunk 1]
```diff
@@ -273,7 +280,8 @@ class CudaPackage(BasePackage):
     def __init__(self, key, versions, name=None):
         super(CudaPackage, self).__init__(key, versions, name)
         if not isinstance(versions, dict):
-            raise TypeError("versions argument should by dict type [cuda_version : list_of_versions")
+            raise TypeError("versions argument should by dict type [cuda_version :" +
+                            "list_of_versions")
```
Comment (mzient) [inline qa/setup_packages.py:284]: Missing space and an unnecessary concatenation
```suggestion
            raise TypeError("versions argument should by dict type [cuda_version : "
                            "list_of_versions")
```

File: qa/setup_packages.py [hunk 2]
```diff
@@ -320,7 +328,8 @@ class CudaPackageExtraIndex(CudaPackage):
     def __init__(self, key, versions, name=None, extra_index=""):
         super(CudaPackageExtraIndex, self).__init__(key, versions, name)
         if not isinstance(versions, dict):
-            raise TypeError("versions argument should be a dictionary {cuda_version_str : list_of_versions}")
+            raise TypeError("versions argument should be a dictionary" +
+                            " {cuda_version_str : list_of_versions}")
```
Comment (mzient) [inline qa/setup_packages.py:332]:
```suggestion
            raise TypeError("versions argument should be a dictionary"
                            " {cuda_version_str : list_of_versions}")
```

File: qa/setup_packages.py [hunk 3]
```diff
@@ -410,72 +420,91 @@ def get_pyvers_name(self, url, cuda_version):
         """
         if isinstance(p.get_supported()[0], tuple):
             # old PIP returns tuple
-            for py_ver in [(x, y, z) for (x, y, z) in p.get_supported() if y != 'none' and 'any' not in y]:
+            for py_ver in [(x, y, z) for (x, y, z) in p.get_supported() if y != 'none' and
+                           'any' not in y]:
```
Comment (mzient) [inline qa/setup_packages.py:424]: Nitpick:
```suggestion
            for py_ver in [(x, y, z) for (x, y, z) in p.get_supported()
                           if y != 'none' and 'any' not in y]:
```

File: qa/setup_packages.py [hunk 4]
```diff
@@ -410,72 +420,91 @@ def get_pyvers_name(self, url, cuda_version):
...
                 py_ver = "-".join(py_ver)
                 ret = self.test_request(url.format(platform=py_ver, cuda_v=cuda_version))
                 if ret:
                     return ret
         else:
             # new PIP returns object
-            for py_ver in [tag for tag in p.get_supported() if tag.abi != 'none' and tag.platform != 'any']:
+            for py_ver in [tag for tag in p.get_supported() if tag.abi != 'none' and
+                           tag.platform != 'any']:
```
Comment (mzient) [inline qa/setup_packages.py:432]: Nitpick:
```suggestion
            for py_ver in [tag for tag in p.get_supported()
                           if tag.abi != 'none' and tag.platform != 'any']:
```

Comment (JanuszL) [reply qa/setup_packages.py]: Done

File: qa/setup_packages.py (hunk 3 above)
Comment (JanuszL) [reply qa/setup_packages.py]: Done

File: qa/setup_packages.py (hunk 2 above)
Comment (JanuszL) [reply qa/setup_packages.py]: Done

**Review by mzient — APPROVED** [review]

**Comment by JanuszL** [comment]
!build

**Review by prak-nv — APPROVED** [review]

**Comment by JanuszL** [comment]
!build

**Pull request merged**